spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Attention.scala

```diff
@@ -43,11 +43,11 @@ class Attention[T: ClassTag](
 
     // Layers for linearly projecting the queries, keys, and values.
     val queryLayer = TransformerOperation.dense(
-      hiddenSize, hiddenSize, false, name = "q").inputs(inputX)
+      hiddenSize, hiddenSize, false, name = s"${this.getName()}_q").inputs(inputX)
```

Comment:
why the change ?

Reply:
a model may have multi attention layers. If names of two layers are same, `getParametersTable` will only get one.